lib/chef/knife/vsphere_vm_network_add.rb

```diff
@@ -0,0 +1,64 @@
+require 'chef/knife'
```

Comment:
hmm... I wonder if there was a Rubocop upgrade. I don't see any value in following this convention. Just ignore it and I'm sorry for the hassle.